packages/web-components/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTRadioGroup } from '@microsoft/fast-foundation';
+
+/**
+ * The base class used for constructing a fluent-radio-group custom element
+ * @public
+ */
+export class RadioGroup extends FASTRadioGroup {
+  /**
+   * Describes the ID of the element that labels the current element, as used for accessibility with screen readers.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: aria-labelledby
+   */
+  @attr({ attribute: 'aria-labelledby' })
+  public ariaLabelledby: string = '';
```

Comment:
This exists on HTML element, is there a reason we need to observe it given we aren't doing anything with it?